syz-manager/html.go

```diff
@@ -349,6 +349,10 @@ func (mgr *Manager) httpCoverCover(w http.ResponseWriter, r *http.Request, funcF
 	}
 
 	do := rg.DoHTML
+	if r.FormValue("debug") != "" {
```

Comment:
Nit: I wonder if we should change the signature of the methods instead. Either add one more debug parameter to put all (now 4) parameters into some struct.

Reply:
Done.